change/@azure-msal-common-c0850675-6450-475c-b9fe-8bc4074f8ffd.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Looks like the changes to common are only package-lock updates?

Reply:
Reset the package-lock and removed the changefile